src/components/Combobox/utils/constants.js

```diff
@@ -1,13 +1,23 @@
 import PropTypes from 'prop-types';
 
-
 export const itemType = PropTypes.shape({
+  icon: PropTypes.shape({
+    icon: PropTypes.string.isRequired,
```

Comment:
all members of `shape` are required by default, so no need to state it explicitly

Reply:
Looks like they changed behavior: https://github.com/facebook/react/issues/5916